go/teams/member_test.go

```diff
@@ -1215,14 +1215,14 @@ func TestMemberCancelInviteEmail(t *testing.T) {
 	}
 	assertInvite(tc, name, address, "email", keybase1.TeamRole_READER)
 
-	if err := CancelEmailInvite(context.TODO(), tc.G, name, address); err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, CancelEmailInvite(context.TODO(), tc.G, name, address, false))
+	require.NoError(t, CancelEmailInvite(context.TODO(), tc.G, name, address, true))
+	require.Error(t, CancelEmailInvite(context.TODO(), tc.G, name, address, false))
```

Comment:
i like this test. can we add explanations to it (or a wee comment) for what it would mean if this test were to fail? like...

```suggestion
	require.NoError(t, CancelEmailInvite(context.TODO(), tc.G, name, address, true), "doesnt error when canceling a canceled invite with allowInaction")
	require.Error(t, CancelEmailInvite(context.TODO(), tc.G, name, address, false), "errors when canceling a canceled invite without allowInaction")
```